### DISCUSSION TITLE
max_heap_table_size & tmp_table_size always ask to increase parameters.

### DISCUSSION THREAD

**Discussion by StaffNowa** [Releem/releem-agent#273] [Feature Requests]
We need by some parameters identify that something terrible with SQL indexes because parameters cannot increase and increase while RAM space is over.

**Comment by drupaladmin** [comment]
Thanks for suggestion.

Yeah, in that case Releem will try to increase memory usage until it possible and safe for the system.

I think we could add more information to Health Check - [Temporary Disk Data](https://releem.com/blog/mysql-health-checks#rec565309868) and to the same tooltip in the Releem Dashboard
 
And our upcoming SQL Query Optimization tool should help here.
